olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -186,7 +187,8 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    * @param data_request The `DataRequest` instance that contains a complete set
    * of request parameters.
    * @note The `GetLayerId` value of the \c DataRequest object is ignored, and
-   * the parameter from the constructor is used instead.
+   * the parameter from the constructor is used instead. CacheWithUpdate fetch
+   * option is not suported.
```

Comment:
The GetLayerId line is not valid anymore, we removed those APIs.
Please remove that and write only about the fetch option:
"@note FetchOptions::CacheWithUpdate is not supported and will result in an error if used for VersionedLayerClient."

Same for below.